src/open_inwoner/js/components/modal/index.js

```diff
@@ -53,13 +63,41 @@ export default class Modal {
     this.text.innerText = text
   }
 
+  setModalIcons(modalIcons) {
+    // Whether the modal-buttons should have icons or not
+    if (modalIcons) {
+      this.node.classList.add('show-modal-icons')
+    }
+  }
```

Comment:
I note these functions only handle the truthy case (eg: don't remove/unset).

Reply:
Yeah this had terrible and sad, lingering effects, so I changed that in the latest commit.